files/common/usr/lib/systemd/scripts/nfs-version-init.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 by Delphix. All rights reserved.
```

Comment:
This copyright should be changed to follow the Apache convention (right?), which you used for the override file.

Reply:
yep. I'll update it